lib/engine/game/g_1867/step/buy_company_preloan.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/buy_company'
+
+module Engine
+  module Game
+    module G1867
+      module Step
+        class BuyCompanyPreloan < Engine::Step::BuyCompany
+          def auto_actions(entity)
+            # If the entity has no loans then passing here makes no difference
+            [Engine::Action::Pass.new(entity)] if entity.loans.empty?
```

Comment:
better to do like 

return [] if entity.loans.empty?
or return nil if ...

Reply:
wouldn't that be the opposite of the intended behavior?

Reply:
i meant the style

return [] unless empty?